### ISSUE TITLE
Exception handing?

### ISSUE THREAD

**Issue by apryiomka** [DuckMan435/PortfolioViewer#6]
How would you handle exceptions for the portfolio API, for instance, SQL Exceptions?

**Comment by DuckMan435** [comment]
I would catch the exception and throw a new instance of HttpResponseException, so that the user would not see all the details involved with the error.  It would make sense to have some internal logging so the team can debug the issue.

**Comment by apryiomka** [comment]
Would you update the code to demonstrate it?

**Comment by DuckMan435** [comment]
Yes, I can update my code for all of the issues you have raised.  When do you need to have the updated commits by?

**Comment by apryiomka** [comment]
No rush, take your time. I am looking at the quality, not the timing or quantity, if you need more time to do it right, take more time. If you can achieve the same result with fewer lines of code, it is much better. You don't have to do it for all controllers / actions, i just would want to see the concept.

**Comment by DuckMan435** [comment]
Thank you, this week is a bit difficult for me, but I will be able to look at it on Sunday.

**Comment by apryiomka** [comment]
Still no exception handling?

**Comment by DuckMan435** [comment]
I will take care of this tonight, my last commit was focused on bringing calculations server side and unit testing.

**Comment by DuckMan435** [comment]
Added Exception Handling and Logging
